src/lib/re.js

```diff
@@ -344,7 +344,7 @@ var $builtinmodule = function (name) {
             if (start == "^") {
                 start = str.indexOf("\n") + 1;
             }
-            if (end == Sk.builtin.none.none$) {
+            if (end === Sk.builtin.none.none$ || endpos === Sk.builtin.none.none$) {
                 end = str.length;
```

Comment:
Is it possible for `end` to be `Sk.builtin.none.none$` here? since it was remapped to Js...

Reply:
Oh, I cant's make sure it could be...
I do the least of change for the code..
if `end` not be `Sk.builtin.none.none$`, we can delete it...

Reply:
I think we can just delete `end === Sk.builtin.none.none$` since it was remapped to js a few lines earlier.

Reply:
Yes, directly return `null`, I changed the condition and committed it.